UNRELEASED.md

```diff
@@ -0,0 +1,4 @@
+**Added:**
+
+- `bpk-component-graphic-promotion`: `3.1.2` => `3.1.3`
+  - Added optional property `contentId`, to apply a HTML attribute `id` to the `div` containing the main content of the Graphic Promotion.
```

Comment:
Out of interest what is the reasoning for naming the property `contentId` over just `id`? 

Given there is only one id applied it would make sense to name it as the default?

Reply:
Perhaps I'm overthinking it but my reasoning here is that my expectation for a `id` is that it would be applied at the top level, whereas this `contentId` is applied nested further in.

I'm happy to rename this to `id` if that works better though